index.js

```diff
@@ -1064,14 +1126,13 @@ fileType.stream = readableStream => new Promise((resolve, reject) => {
 	});
 });
 
-Object.defineProperty(fileType, 'extensions', {
-	get() {
-		return new Set(supported.extensions);
-	}
-});
-
-Object.defineProperty(fileType, 'mimeTypes', {
-	get() {
-		return new Set(supported.mimeTypes);
-	}
-});
+module.exports = {
+	fromStream,
+	fromTokenizer,
```

Comment:
I don't think we should publicly expose the tokenizer.

Reply:
I deliberately publicly exposed `fromTokenizer`. It allows to use specialized tokenizers like [tokenizer-http](https://github.com/Borewit/tokenizer-http) (HTTP-client using HTTP range requests). The same technology I used in [music-metadata-s3](https://github.com/Borewit/music-metadata-s3), to read very efficient from the Amazon S3 cloud. I am working on generic S3-tokenizer. 

Why not using a stream? This interface propagates the ignore calls, which can be converted in a very efficient requests of only the data really required, such as the first 384 bytes of the file, and the last 2k.

If you find it distracting, I will remove it, no problem. But it was exposed on purpose.

Reply:
Thanks for elaborating. It does make sense to expose it, but it needs to be documented what it's for.

Reply:
Done here: #279